applications/tari_collectibles/src-tauri/src/clients/wallet_client.rs

```diff
@@ -117,12 +117,11 @@ impl WalletClient {
     merkle_root: Vec<u8>,
   ) -> Result<grpc::CreateInitialAssetCheckpointResponse, CollectiblesError> {
     let inner = self.get_inner_mut()?;
-    let committee = vec![];
     let request = grpc::CreateInitialAssetCheckpointRequest {
       // TODO: contract id
       contract_id: Vec::from_hex(asset_public_key)?,
       merkle_root,
-      committee,
+      committee_signatures: None,
```

Comment:
I take it this is still test/initial code?
Surely the initial checkpoint needs to be signed by the committee?

Reply:
yeah this is in the collectibles app, which is a little in limbo at the moment